src/core/Akka/Actor/ExtendedActorSystem.cs

```diff
@@ -27,6 +27,10 @@ public abstract class ExtendedActorSystem : ActorSystem
         /// </summary>
         public abstract IInternalActorRef Guardian { get; }
 
+        /// <summary>
+        /// The <see cref="RootGuardianActorRef"/>, used as the lookup for <see cref="IActorRef"/> resolutions.
+        /// </summary>
+        public abstract IInternalActorRef LookupRoot { get; }
```

Comment:
Needed to expose the `RootGuardianActorRef` on the `ActorSystemImpl` so it could be accessed from `ActorLookupSpecs`.